index.html

```diff
@@ -95,9 +95,9 @@ <h2>Tonic</h2>
             </section>
         </section>
 
-        <section class="project-section">
+        <section class="project-section" id="availibity-card">
             <figure>
-                <img src="./Images/Availibity.png" alt="" id="snap-shot-image">
+                <img src="./Images/Availibity.png" alt="" class="snap-shot-image">
             </figure>
```

Comment:
- [ ] While you make a great effort 😃I see that you forgot to add the background color behind the sections to make it like the Figma Design 😊.so it will be better if you add the background color for the body behind the section of the projects 🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/235673323-5fa1c2a6-5280-4126-abb9-65f523d4ba9a.png)   |   ![image](https://user-images.githubusercontent.com/83360636/235673140-9a53f18d-7679-4bce-9854-66e6dd6910fc.png)   |